mods/cnc/maps/nod06a/map.yaml

```diff
@@ -0,0 +1,729 @@
+	waypoint10: waypoint
+		Location: 31,41
+		Owner: Neutral
+	waypoint9: waypoint
+		Location: 19,36
+		Owner: Neutral
+	waypoint8: waypoint
+		Location: 20,54
+		Owner: Neutral
+	waypoint7: waypoint
+		Location: 40,57
+		Owner: Neutral
+	waypoint6: waypoint
+		Location: 36,29
+		Owner: Neutral
+	waypoint5: waypoint
+		Location: 40,29
+		Owner: Neutral
+	waypoint4: waypoint
+		Location: 44,44
+		Owner: Neutral
+	waypoint3: waypoint
+		Location: 44,47
+		Owner: Neutral
+	waypoint2: waypoint
+		Location: 59,39
+		Owner: Neutral
+	waypoint1: waypoint
+		Location: 51,43
+		Owner: Neutral
+	waypoint0: waypoint
+		Location: 56,39
+		Owner: Neutral
+	UnitsEntryLeft: waypoint
+		Location: 20,17
+		Owner: Neutral
+	UnitsRallyLeft: waypoint
+		Location: 20,20
+		Owner: Neutral
+	UnitsEntryRight: waypoint
+		Location: 27,17
+		Owner: Neutral
+	UnitsRallyRight: waypoint
+		Location: 27,20
+		Owner: Neutral
+	ChnEnrty: waypoint
+		Location: 32,61
+		Owner: Neutral
+	Obj2UnitsEntry: waypoint
+		Location: 48,17
+		Owner: Neutral
+	Guard1: e2
+		Location: 18,31
+		Owner: GDI
+		SubCell: 0
+	Guard2: e2
+		Location: 17,31
+		Owner: GDI
+		SubCell: 1
+	Guard3: e2
+		Location: 18,32
+		Owner: GDI
+		SubCell: 2
+	Atk1Activator1: gtwr
+		Location: 48,43
+		Owner: GDI
+	Atk1Activator2: e1
+		Location: 50,46
+		Owner: GDI
+		Facing: 224
+		SubCell: 4
+	Atk2Activator1: gtwr
+		Location: 51,46
+		Owner: GDI
+	Atk2Activator2: e1
+		Location: 48,44
+		Owner: GDI
+		Facing: 160
+		SubCell: 4
+	Atk3Activator: v25
+		Location: 41,19
+		Owner: GDI
+	Chn1Activator1: v29
+		Location: 42,22
+		Owner: GDI
+	Chn1Activator2: v27
+		Location: 35,23
+		Owner: GDI
+	Chn1Activator3: v26
+		Location: 37,20
+		Owner: GDI
+	Chn1Activator4: v24
+		Location: 41,23
+		Owner: GDI
+	Chn1Activator5: v22
+		Location: 41,21
+		Owner: GDI
+	Chn2Activator1: v27
+		Location: 34,23
+		Owner: GDI
+	Chn2Activator2: v30
+		Location: 35,22
+		Owner: GDI
+	Chn2Activator3: v27
+		Location: 36,23
+		Owner: GDI
+	Detonator: CRATE.plain
+		Location: 59, 43
+		Owner: GDI
+	Radar: hq
+		Location: 57,32
+		Owner: GDI
+		
+Smudges:
+
+Rules:
+	Player:
+		-ConquestVictoryConditions:
+		MissionObjectives:
+			EarlyGameOver: true
+		EnemyWatcher:
+	World:
+		-CrateSpawner:
+		-SpawnMPUnits:
+		-MPStartLocations:
+		ObjectivesPanel:
+			PanelName: MISSION_OBJECTIVES
+		LuaScript:
+			Scripts: nod06a.lua
+	^Vehicle:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Tank:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false		
+	^Infantry:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+		AnnounceOnSeen:
+	^Plane:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Ship:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Building:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^Husk:
+		Tooltip:
+			GenericVisibility: Enemy
+			ShowOwnerRow: false
+	^CivBuildingHusk:
+		Tooltip:
+			GenericVisibility: Enemy
```

Comment:
Sorry, the comment above wasn't completely right. The `GenericVisibility` line needs to go away for the tooltip to be correct. (only for the `^CivBuildingHusk`, the other one is ok.)